packages/gatsby-remark-images/src/index.js

```diff
@@ -103,12 +100,17 @@ module.exports = (
     const fileNameNoExt = fileName.replace(/\.[^/.]+$/, ``)
     const defaultAlt = fileNameNoExt.replace(/[^A-Z0-9]/gi, ` `)
 
-    // TODO
-    // Fade in images on load.
-    // https://www.perpetual-beta.org/weblog/silky-smooth-image-loading.html
-
-    const imageClass = `gatsby-resp-image-image`
-    const imageStyle = `width: 100%; height: 100%; margin: 0; vertical-align: middle; position: absolute; top: 0; left: 0; box-shadow: inset 0px 0px 0px 400px ${
+    const imageStyle = `
+      width: 100%;
+      height: 100%;
+      margin: 0;
+      vertical-align: middle;
+      position: absolute;
+      top: 0;
+      left: 0;
+      transition: opacity 0.5s;
+      transition-delay: 0.5s;
+      box-shadow: inset 0px 0px 0px 400px ${
```

Comment:
Was making this verbose intentional? This inflates outputted html with extra bytes. Perhaps we can keep it in readable form but actually output compressed code?

Reply:
Sure, my assumption was that this was minified down so shouldn't make a difference but much easier to understand what's going on.